### PR TITLE
refactor: init casabi controller in init function

### PR DESCRIPTION
**Pull request by CFenner**
preparation for #66

### CODE REVIEW

File: custom_components/casambi/light.py [hunk 1]
```diff
@@ -202,182 +103,181 @@ async def async_setup_entry(
     return True
 
 
-async def async_setup_platform(
```
Comment (CFenner) [inline custom_components/casambi/light.py:205]: @hellqvio86 is this one needed when setting up the integration from `configuration.yaml`?

Comment (hellqvio86) [reply custom_components/casambi/light.py:205]: Yes that's correct, it has been supported for a while so in order to remove it the integration will probably need to fire of a warning that its soon going away.

Comment (hellqvio86) [reply custom_components/casambi/light.py:205]: I have added a warning that the code path will be removed in future version.

**Comment by hellqvio86** [comment]
@CFenner you have gone through most of the code base now 🙂

So just wanted to ask if you are interested in maintainer right to the project? Sorry for the message being public, but I didn't find an email address.

**Comment by CFenner** [comment]
> @CFenner you have gone through most of the code base now 🙂

Yes, in deed 😉 but `aiocasmbi` is still not that clear to me. To have maintainer permissions would be great as I plan to further use the integration (if the API will still be available) and at least implement #54 for my need.

btw: email address is in the git log.

File: custom_components/casambi/light.py (hunk 1 above)
Comment (CFenner) [reply custom_components/casambi/light.py:205]: Added code again but extracted the code to reuse.

**Review by hellqvio86 — APPROVED** [review]
lgtm, thanks for the pull request! :slightly_smiling_face:

**Pull request merged**